### DISCUSSION TITLE
Getting the first data signal then nothing. What's going on?

### DISCUSSION THREAD

**Discussion by jasonjgeiger** [universam1/iSpindel#576] [Q&A]
When connecting it to a service for transferring data I get the first data signal only and then nothing after. While connected directly to the iSpindel the info updates consistently till I manually disconnect from it.

No matter what configuration I use I can confirm **data is transferring to the service but only the first interval, but nothing after**. 
    - I've reset the iSpindel multiple times
    - Connected to both my wifi networks
    - I've tried different intervals 120s and up 15m
    - I've tried with a local MQTT server and Ubidots
    - I'm running the latest firmware 7.1.4

I haven't seen anyone having a similar issue. Any suggestions?

**Comment by ErikdBr** [comment]
This is actualy a common issue and could be anythhing from a faulty d1 mini to resistor instead of diode to dodgy soldering. 

https://github.com/universam1/iSpindel/issues?q=is%3Aissue+send+data+once
